src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -348,6 +348,28 @@ private boolean compatibleOptions(Options oldOptions, Options newOptions) {
             return false;
         }
 
+        // Libraries loaded during pre-initialization
+
+        if (newOptions.PATCHING != oldOptions.PATCHING) {
+            Log.LOGGER.fine(notReusingContext + "loading patching is " + newOptions.PATCHING);
```

Comment:
What will this print? `loading patching is true` that's not very clear. How about `loading patching enabled` or something like that?

Reply:
I was thinking to use a helper function to log this in a more generic way:
```
not reusing pre-initialized context: -X<option> is <new_value> (was <old_value>)
```
as it's easy to forget to update one of the 3 or 4 occurrences of the option in this manual handling.

Reply:
Logger helper functions have a problem - they ruin the meta information you get in the logs about where the warning came from.

Reply:
Well it means people don't know where to look to find the error. I won't object to a couple of cases, but we should not be in the habit of breaking this functionality of logging.